main.go

```diff
@@ -78,7 +83,13 @@ func main() {
 		log.Infof("Autodiscovered Base Role ARN: %s", config.BaseRoleARN)
 	}
 
-	credsGetter := NewSTSCredentialsGetter(awsSess, config.BaseRoleARN)
+	awsConfigs := make([]*aws.Config, 0)
```

Comment:
Did you mean for this to be `make([]*aws.Config, 0, 1)`? That is, anticipate space for one, but sometimes only use zero?

Reply:
Done